### DISCUSSION TITLE
Reauthentication in Github Desktop

### DISCUSSION THREAD

**Discussion by timothydilbert** [community/community#141597] [Enterprise]
### Select Topic Area

Question

### Body

We use Github Desktop to connect to repos within our Github Enterprise SaaS subscription. I've noticed that, after the initial setup and authentication, developers are never asked to re-authenticate in Github Desktop. I'm a little nervous about this configuration.

Instead, I'd like Github Desktop to ask developers to re-authenticate every 24HRs. And, if Github Desktop does not support this, does anyone else have this anxiety? And, what application or workaround did you implement?

**Comment by queenofcorgis** [comment]
Hi there! It doesn't look like there’s a way right now. It installs as an app so as long as your account is enabled, it will continue to work. It, like all other tokens and personal apps, are disabled when your account is disabled assuming you’re using EMU. If you have a traditional enterprise, removing a user from the org won’t invalidate their auth but it will remove their access to any repos from that org.
